ext/stringio/stringio.c

```diff
@@ -615,11 +669,12 @@ strio_to_read(VALUE self)
 
 /*
  * call-seq:
- *   strio.eof     -> true or false
- *   strio.eof?    -> true or false
+ *   eof? -> true or false
+ *
+ * Returns +true+ if positioned at end-of-stream +false+ otherwise;
```

Comment:
Minor punctuation nit:

```suggestion
  * Returns +true+ if positioned at end-of-stream; +false+ otherwise;
```

Reply:
Added comma (consistent w/others).